python/plugins/processing/algs/qgis/postgis_utils.py

```diff
@@ -153,7 +153,7 @@ def __init__(self, host=None, port=None, dbname=None, user=None,
         try:
             self.con = psycopg2.connect(self.con_info())
         except psycopg2.OperationalError as e:
-            raise DbError(unicode(e))
+            raise DbError(str(e))
```

Comment:
https://github.com/arnaud-morvan/QGIS/blob/processing_postgis_errors/python/plugins/processing/algs/qgis/postgis_utils.py#L100:L101

and 
```
"test %s" % unicode("testé")
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 4: ordinal not in range(128)
```
IMHO, as far as error message are threated as str, it should be passed as str.
Error messages should keep simple and display correctly on all type of devices, no need for special characters.